tests/emails.test.js

```diff
@@ -0,0 +1,104 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const mongoose = require('mongoose');
+const {server} = require('../src/app');
+chai.should();
+chai.use(chaiHttp);
+
+describe("POST Tests Emails",()=>{
+    it("POST /emails Not Found", (done) =>{
+        const email = {
+            email: "martha_13_7@gmail.com"
+        };
+        chai.request(server).post("/emails").send(email)
+        .end( (error, response) => {
+            response.should.have.status(404);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("No se encontro la cuenta");
+            done();
+        });
+     });
+
+    it("POST /emails Bad Request", (done) =>{
+        const email = {
+            email: 1223
+        };
+        chai.request(server).post("/emails").send(email)
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.be.a("object");
+            response.body.should.have.property("errors");
+            done();
+        });
+    });
+
+    it("POST /emails", (done) =>{
+        const email = {
+            email: "martha_15_7@outlook.com"
+        };
+        chai.request(server).post("/emails").send(email)
+        .end( (error, response) => {
+            response.should.have.status(201);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("El código de confirmación se reenvio exitosamente");
+            done();
+        });
+     });
+});
+
+
+describe("POST Tests Emails Accounts",()=>{
+    let accessToken= null;
+
+    before((done) =>{
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
    before((done) => {
```